### DISCUSSION TITLE
v1.3.6.0 Release - CPU Usage Runaway Fix

### DISCUSSION THREAD

**Discussion by RadzPrower** [ajohns6/Zelda-3-Launcher#40] [Announcements]
- Made a change to closing settings window behavior so that it is consistent between the cancel button and the close (X) button. (#37)
- Fixed runaway CPU usage while `zelda3.exe` was running due to it waiting on the game to close and resume control. (#39)

**Full Changelog**: https://github.com/RadzPrower/Zelda-3-Launcher/compare/v1.3.5.0...v1.3.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/RadzPrower/Zelda-3-Launcher/releases/tag/v1.3.6.0'>v1.3.6.0 Release - CPU Usage Runaway Fix</a>.</em>

**Comment by Pepe031984** [comment]
Hello

Is there also a German version?

**Reply by RadzPrower** [reply]
There is supposedly German support and I supported it as much as I could, but that comes from the main project rather than my GUI launcher.

**Reply by Pepe031984** [reply]
I tried to open this file with the Zelda 3 launcher but I get the error. 

[Legend of Zelda, The - A Link to the Past (Germany).zip](https://github.com/user-attachments/files/20062187/Legend.of.Zelda.The.-.A.Link.to.the.Past.Germany.zip)

![1](https://github.com/user-attachments/assets/ca2e52f9-77f5-4d1c-be9b-861d3628260e)

**Reply by RadzPrower** [reply]
Yeah, I did not pick the translation patches. You'll have to use whatever one is listed in the original snearev project so that the checksums match.

**Reply by Pepe031984** [reply]
What exactly do I have to do?

**Reply by RadzPrower** [reply]
Find the correct patch so the checksums match. I do not follow the translation scenes, so I don't actually know which one or where.

Again, I did not make that call.

**Reply by Pepe031984** [reply]
Who can I ask?

**Reply by RadzPrower** [reply]
snesrev, but they've pretty much gone dark, so best bet would probably be the Discord for snesrev's original project.

**Reply by Pepe031984** [reply]
OK Thanks

**Reply by KentEchardt** [reply]
How can I map the turbo button to my controller? I tried but it's not working.

**Comment by frinceleynandj-source** [comment]
help i can download source code but not the actual file

**Reply by RadzPrower** [reply]
Where exactly are you trying to download from?
